sherpa/csrc/online-transducer-fast-beam-search-decoder.cc

```diff
@@ -127,9 +127,7 @@ void OnlineTransducerFastBeamSearchDecoder::Decode(
       ++t;
       ++p->num_trailing_blanks;
       continue;
-    } else {
-      p->num_trailing_blanks = 0;
-    }
+    p->num_trailing_blanks = 0;
```

Comment:
Sorry. You removed the right `}`. Could you add it?

Reply:
Sorry my bad, just committed your suggestion without checking. Fixed it!